Framework/CurveFitting/src/Algorithms/PlotPeakByLogValue.cpp

```diff
@@ -540,6 +541,22 @@ std::string PlotPeakByLogValue::getMinimizerString(const std::string &wsName,
   return format;
 }
 
+std::vector<std::vector<double>>
+PlotPeakByLogValue::getExclude(size_t numSpectra) {
+  std::vector<double> excludeList = getProperty("Exclude");
+  std::vector<std::vector<double>> excludeVector;
+  for (size_t i = 0; i < numSpectra * 2; i += 2) {
+    if (i < excludeList.size()) {
+      std::vector<double> excludePair = {excludeList[i], excludeList[i + 1]};
+      excludeVector.push_back(std::vector<double>(excludePair));
```

Comment:
Can we use `emplace_back` to avoid creating a copy of the `std::vector<double>`?